ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/RaoResult.java

```diff
@@ -24,8 +25,16 @@
         SUCCESS
     }
 
+    public enum LastSensiParams {
```

Comment:
Same, I think the name is not explicit enough

Reply:
What do you suggest?

Reply:
SystematicSensitivityAnalysisParameters ?

Reply:
But actually this is quite weird RaoResult must be generic to all Rao results but this information is really related to LinearRao. Can't we do a LinearRaoResult which is an extension of RaoResult ? @baptsegu

Reply:
yes we can. And I think we should. Anyway I know @terriervik loves extensions (sorry :D)